lib/resty/dns/resolver.lua

```diff
@@ -65,6 +69,10 @@ local _M = {
     TYPE_SRV    = TYPE_SRV,
     TYPE_SPF    = TYPE_SPF,
     CLASS_IN    = CLASS_IN,
+    SECTION_QD  = SECTION_QD,
```

Comment:
This `SECTON_QD` constant is unused internally in the implementation nor documented in README. Better remove it?